File: Solution in CPP/Que9.cpp
// Reverse a string 
#include<bits/stdc++.h>
using namespace std;

string reverseString(string s){
    string result = "";
    for(int i = s.length() - 1; i>= 0; i--){
        result += s[i];
    }

    return result;
}

int main(){
    string s = "Aniket";
    string ans = reverseString(s);
    cout<<"Reverse = "<<ans<<endl;
}

File: Solution in RUST/Que9.rs
// Reverse a string in Rust

fn reverse_string(s: &str) -> String {
    let mut result = String::new();
    for ch in s.chars().rev() {
        result.push(ch);
    }
    result
}

fn main() {
    let s = "Aniket";
    let ans = reverse_string(s);
    println!("Reverse = {}", ans);
}
